build-images.sh

```diff
@@ -313,6 +313,16 @@ popd
 # Append the image URL to the images array
 images+=("${repobase}/${reponame}")
 
+#############################
+##      FlexiSIP proxy     ##
+#############################
+echo "[*] Build flexisip container"
+reponame="nethvoice-flexisip"
+pushd flexisip
+buildah build --force-rm --layers --jobs "$(nproc)" --target production --tag "${repobase}/${reponame}"
```

Comment:
```suggestion
buildah build --force-rm --layers --jobs "$(nproc)" --target production --squash --tag "${repobase}/${reponame}"
```

This shrinks all the layers into one, we're not reusing them anyway